### PR TITLE
Remove the CoC acceptance functionality and add slash commands

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
Related issue: #13 

## Checklist:

- [X] Code Quality
- [x] Pep-8
- [X] Tests (if applicable)
- [X] Success Criteria Met
- [X] Inline Documentation
- [X] Wiki Documentation (if applicable)

### CODE REVIEW

File: buttercup/strings/en_US.yaml [hunk 1]
```diff
@@ -26,4 +18,4 @@ restrictor:
   wrong_prefix: |
     Hey <@{0}>, it appears that you have attempted to change your nickname, but forgot the initial `/` in your nickname. Make sure that your nickname starts with `/u/` and not `u/`!
   new_lifted: |
-    Welcome <@{0}> to the Transcribers of Reddit Discord! Since you have accepted our Code of Conduct and changed your nickname to the correct format, you now have unrestricted access to our Discord. Thanks for your time and enjoy your time in here!
\ No newline at end of file
+    Welcome <@{0}> to the Transcribers of Reddit Discord! Since you have changed your nickname to the correct format, you now have unrestricted access to our Discord. Thanks for your time and enjoy your time in here!
```
Comment (TimJentzsch) [inline buttercup/strings/en_US.yaml:21]: This is missing a new line at the end of the file

**Review by TimJentzsch — CHANGES_REQUESTED** [review]
Looks very good! Just missing a new line at the end of a file.

File: buttercup/strings/en_US.yaml (hunk 1 above)
Comment (MaxVanDeursen) [reply buttercup/strings/en_US.yaml]: I have added this newline!

**Review by TimJentzsch — APPROVED** [review]

File: buttercup/strings/en_US.yaml (hunk 1 above)
Comment (TimJentzsch) [reply buttercup/strings/en_US.yaml]: Thanks, looks good now!

**Pull request merged**